src/Backend/Form/Type/MetaType.php

```diff
@@ -242,34 +294,34 @@ private function getMetaReverseTransformFunction(): callable
 
             if ($metaId === null || !$this->meta[$metaId] instanceof Meta) {
                 return new Meta(
-                    $metaData['keywords'],
-                    $metaData['keywordsOverwrite'],
-                    $metaData['description'],
-                    $metaData['descriptionOverwrite'],
-                    $metaData['title'],
-                    $metaData['titleOverwrite'],
-                    $metaData['url'],
-                    $metaData['urlOverwrite'],
+                    $metaData['keywords'] ?? '',
+                    $metaData['keywordsOverwrite'] ?? false,
+                    $metaData['description'] ?? '',
+                    $metaData['descriptionOverwrite'] ?? false,
+                    $metaData['title'] ?? '',
+                    $metaData['titleOverwrite'] ?? false,
+                    $metaData['url'] ?? '',
+                    $metaData['urlOverwrite'] ?? false,
                     $metaData['custom'] ?? null,
-                    SEOFollow::fromString((string) $metaData['SEOFollow']),
-                    SEOIndex::fromString((string) $metaData['SEOIndex']),
+                    SEOFollow::fromString((string)$metaData['SEOFollow']),
+                    SEOIndex::fromString((string)$metaData['SEOIndex']),
```

Comment:
please restore the spaces after the casts and go through your PR to find the other places as well. 
Might be best to fix the code styles of your IDE